test/common/storagelib.py

```diff
@@ -87,7 +87,8 @@ def add_loopback_disk(self, size=50, name=None):
         # machine, they might come back for unknown reasons, in a
         # non-functional state. Running partprobe will get rid of
         # them.
-        self.machine.execute("partprobe '%s'" % dev)
+        # See https://github.com/cockpit-project/bots/pull/5793 for why `|| true`
+        self.machine.execute("partprobe '%s' || true" % dev)
```

Comment:
If this was more than just an experiment, please restrict this to ubuntu-2204. This makes things brittle and hard to debug, so let's restrict the blast radius. Thanks!

Reply:
It's not enough anyway — this masks the partprobe failure, but other partitioning tools just fail later on.  I'm pretty annoyed that I didn't get a better downstream response on this...